python/HelloWorldTest.py

```diff
@@ -22,52 +19,48 @@ class HelloWorldTests (unittest.TestCase):
     MESSAGE_TEXT_CSS = "div#et_pb_contact_form_1 div.et-pb-contact-message"
 
     def setUp(self):
-        # It assumes chrome driver is saved at <user home>/drivers/chromedriver
-        user_home = str(Path.home())
-        driver_path = r"drivers"
-        driver_name = r"chromedriver.exe" if _platform == "Windows" else r"chromedriver"
-        webdriver_path = os.path.join(user_home, driver_path, driver_name)
         chrome_options: Options = webdriver.ChromeOptions()
```

Comment:
This is already removed from the latest version.

Reply:
This is actually most problematic area for beginners to start running Selenium tests. (My hope is to prevent as much as possible.) Maybe We can provide a check list Selenium test can't find webdrivers.